android-pay/src/main/java/com/stripe/wrap/pay/utils/CartManager.java

```diff
@@ -53,7 +55,7 @@ public CartManager() {
      * @param currencyCode a currency code used for this cart, and the rest of the application
      */
     public CartManager(String currencyCode) {
-        mCurrency = PaymentUtils.getCurrencyByCodeOrDefault(currencyCode);
+        mCurrency = Currency.getInstance(currencyCode.toUpperCase());
```

Comment:
Reverting to this call also allows a RuntimeException if the user tries to use an invalid currency code. I do force it to be uppercase, so at least it protects against the crash that occurs with `Currency.getInstance("usd")`